src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -29,13 +43,17 @@ public void getStatistic(String fromFileName, String toFileName) {
                 }
                 value = bufferedReader.readLine();
             }
-            int result = supplyAmount - buyAmount;
-            toWrite = String.format("supply,%d%n"
-                    + "buy,%d%n"
-                    + "result,%d%n", supplyAmount, buyAmount, result);
         } catch (IOException e) {
             throw new RuntimeException("Cannot read the file" + fromFileName, e);
         }
+    }
+
+    private void calculateStatistic() {
```

Comment:
Rename the method to createReport.
The method should get the String variable and calculate data and return a report as a String result.